src/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react'
+import ReactDOM from 'react-dom'
+
+const Options = () => {
+    const [webhookUrl, setWebhookUrl] = useState('')
+    const handleChange = (e) => setWebhookUrl(e.target.value) // TODO: handle url validation
```

Comment:
Also adding an bug/issue for URL validation https://github.com/mazzola/dnd-beyond-dice-chrome-extension/issues/13